.ci/.env.prd

```diff
@@ -19,4 +19,8 @@ REACT_APP_AUTH0_DOMAIN=decentraland.auth0.com
 REACT_APP_AUTH0_REDIRECT=https://builder.decentraland.org/callback
 REACT_APP_MANA_TOKEN_CONTRACT_ADDRESS=0x0f5d2fb29fb7d3cfee444a200298f468908cc942
 REACT_APP_LAND_REGISTRY_CONTRACT_ADDRESS=0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d
-REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS=0x959e104e1a4db6317fa58f8295f586e1a978c297
\ No newline at end of file
+REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS=0x959e104e1a4db6317fa58f8295f586e1a978c297
+REACT_APP_MARKETPLACE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/marketplace-ropsten
```

Comment:
For production this var should be `https://api.thegraph.com/subgraphs/name/decentraland/marketplace` (without the `-ropsten` at the end)